ancestrymanager/ancestrymanager.go

```diff
@@ -126,52 +131,26 @@ func (m *onlineAncestryManager) GetAncestry(project string) (string, error) {
 	return path, nil
 }
 
-// GetAncestryWithResource first attempts to get Ancestry from the API
-// If that fails, it falls back to inspecting the resource.
-func (m *onlineAncestryManager) GetAncestryWithResource(project string, tfData resources.TerraformResourceData, cai resources.Asset) (string, error) {
-	path, err := m.GetAncestry(project)
-	if path != "" {
-		return path, nil
-	}
-
-	ancestry, ok := m.getAncestryFromResource(tfData, cai)
-	if !ok {
-		return "", err
-	}
-	path = ancestryPath(ancestry)
-	m.errorLogger.Info(fmt.Sprintf("[Online] Retrieved ancestry for %s: %s", project, path))
-	m.store(project, path)
-	return path, nil
-}
-
-func (m *onlineAncestryManager) store(project, ancestry string) {
-	if project != "" && ancestry != "" {
-		m.ancestryCache[project] = ancestry
-	}
-}
-
-type offlineAncestryManager struct {
-	resourceAncestryManager
-	project  string
-	ancestry string
-}
-
-// GetAncestry returns the ancestry for the project. It returns an error if
-// the project does not equal to the one provided during initialization.
-func (m *offlineAncestryManager) GetAncestry(project string) (string, error) {
-	if project != m.project {
-		return "", fmt.Errorf("cannot fetch ancestry in offline mode")
+func (m *manager) store(project, ancestry string) {
+	if project != "" {
+		if m.offline {
```

Comment:
We shouldn't need to have a separate check for offline vs online mode here because ancestry is required to be non-empty in offline mode.  https://github.com/GoogleCloudPlatform/terraform-validator/blob/21cb55867a0d21110e73c5ff8360b54f68daf48e/cmd/validate.go#L95

(If having a separate check for offline is required to get tests to pass the tests are probably wrong.)

Reply:
Restored this to the original version of online ancestry manager's store method. 
Have to change convert_test.go's newTestConverter() to fill in a non empty string for ancestry to make unit test work.